src/build-board.coffee

```diff
@@ -166,10 +166,10 @@ module.exports = (name, layers = []) ->
     # now build group for the color and the silkscreen (if it exists) and
     # mask away the soldermask holes
     smId = "#{name}-sm_#{uniqueId()}"
-    defs.push { mask: { id: smId, color: '#000', _: [
+    defs.push { mask: { id: smId, color: '#000', _: { g: { _: [
           bboxRect null, '#fff'
           { use: { 'xlink:href': "##{mask}" } }
-        ]
+        ]}}
```

Comment:
Please try to match the indent style for braces and brackets with the rest of the file